_posts/2017-05-01-0.96.0-release.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+category: releases
+title: Armory 0.96.0 Released
+date: 2017-05-01 00:00:00 -0000
+---
+
+This release is a major release introducing many new features, bug fixes, and optimizations.
+
+## Compatibility
+
+Armory 0.96.0 introduces new output script types, P2SH-P2PK and P2SH-P2WPKH. Spending from these output types requires Armory 0.96.0+. The standard P2PKH output types are still created and can be signed for by previous versions dating back to Armory 0.92.x.
+
+This version is also compatible with Bitcoin Core 0.14.0+.
+
+## Notable Changes
+
+### Segregated Witness Support
+
+Armory 0.96.0 contains all of the code necessary in order for it to use Segregated Witness. This functionality is currently hard-coded to be disabled on mainnet, but is enabled on testnet for users to experiment with this feature if they wish.
+
+In preparation for Segwit's activation, two new output script types have been added. Users can now choose between P2PKH, P2SH-P2PK, and P2SH-P2WPKH addresses. While Segwit is still not activated, the P2SH-P2WPKH address type will not be available. P2PKH are the standard addresses that are currently in use. Those addresses are backwards compatible and can be used in older versions of Armory. P2SH-P2PK is a Pay-to-Pubkey script as the redeemscript of a P2SH address. P2SH-P2WPKH is a Pay-to-Witness-Pubkey-Hash output nested within a P2SH output as specified by [BIP 141](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#P2WPKH_nested_in_BIP16_P2SH). In order to spend from P2SH-P2PK and P2SH-P2WPKH outputs, you must use Armory 0.96+.
+
+### Migrating to A New Wallet Format
+
+Armory 0.96.0 marks the beginning to a new wallet format written in C++ and uses LMDB. Currently this wallet format is only a mirrored version of the current Python wallet format. It only contains public data and metadata in order to allow the C++ signing code to function, including support for P2SH-P2PK and P2SH-P2WPKH outputs which are not supported by the current Python wallet format. In the future, this new format will support BIP 32 and compressed public keys.
```

Comment:
mirrored wallets already support compressed pub keys. all the p2sh scripts use compressed keys